apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/FissClaimRdaSink.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.bfd.pipeline.rda.grpc.sink;
+
+import gov.cms.bfd.model.rda.PreAdjFissClaim;
+import gov.cms.bfd.pipeline.rda.grpc.ProcessingException;
+import gov.cms.bfd.pipeline.sharedutils.PipelineApplicationState;
+import java.util.Optional;
+
+public class FissClaimRdaSink extends AbstractClaimRdaSink<PreAdjFissClaim> {
+  public FissClaimRdaSink(PipelineApplicationState appState) {
+    super(appState);
+  }
+
+  @Override
+  public Optional<Long> readMaxExistingSequenceNumber() throws ProcessingException {
+    return readMaxExistingSequenceNumber(
+        String.format(
+            "select max(c.%s) from PreAdjFissClaim c", PreAdjFissClaim.Fields.sequenceNumber));
```

Comment:
isn't the table name "pre_adj"."FissClaims"

Reply:
Hibernate query language uses the entity name rather than the table name in the query.